.gitattributes

```diff
@@ -1,10 +1,8 @@
 .github/ export-ignore
-dist/ export-ignore
 tests/ export-ignore
 .gitattributes export-ignore
 .gitignore export-ignore
 .pylintrc export-ignore
-codecov.yml export-ignore
```

Comment:
I don't understand, does this mean `codecov.yml` should be exported now in zip packages?

Reply:
It think the file is removed from the repo, and the settings are defined on the organisation.

Reply:
Yes, I forgot to clean it up earlier.

Having codecov defined  at the org lrvel has one benefit, it doesn't suddenly fail to work. But if there is a need to have to tweak it, we can still reintroduce it.

Reply:
There is still a codecov.yml in this repo!